packages/m/mathter/xmake.lua

```diff
@@ -7,9 +7,12 @@ package("mathter")
     add_urls("https://github.com/petiaccja/Mathter/archive/refs/tags/$(version).tar.gz",
              "https://github.com/petiaccja/Mathter.git")
 
+    add_versions("v2.0.0", "cec8472d3a56613d4815d1a14e5bf42976884177cc5283c5ad5aba0710cc4bab")
     add_versions("v1.1.2", "9e6d03295d28e8792721fedca5d53955d4057d1550e51491408353b6181e6c6d")
     add_versions("v1.1.1", "510e6aa198cd7b207a44d319e4471021f207cba8c4d2d7e40086f1f042fe13ab")
 
+    add_patches("v2.0.0", "https://github.com/petiaccja/Mathter/pull/55.diff", "6e9de8e11e49c35d32c771ef99dc63bf9cd2aae894187a4f77b00ae54d332b22")
+
```

Comment:
Changes for using the v2.0.1 patch release:

```suggestion
    add_versions("v2.0.1", "7fb6c9c3e16f8dcfd14b6a0f2c333e48211a2e8a1b04e68ddf322f8affee9ef1")
    add_versions("v1.1.2", "9e6d03295d28e8792721fedca5d53955d4057d1550e51491408353b6181e6c6d")
    add_versions("v1.1.1", "510e6aa198cd7b207a44d319e4471021f207cba8c4d2d7e40086f1f042fe13ab")
```

Let's wait for the CI in https://github.com/xmake-io/xmake-repo/pull/7417 to succeed, because the hash might change if adjustments are needed for the v.2.0.1 release.

Reply:
I remember that the patch shasum of github commit is unstable and may change.

If it is a small patch, you can add it directly.